og.module

```diff
@@ -290,3 +296,50 @@ function og_invalidate_group_content_cache_tags(EntityInterface $entity) {
     Cache::invalidateTags(Cache::buildTags('og-group-content', $tags));
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_insert() for user_role entities.
+ */
+function og_og_role_insert(OgRoleInterface $role) {
+
+  $add_id = 'og_membership_add_role_action.' . $role->id();
+  if (!Action::load($add_id)) {
+    $action = Action::create(array(
```

Comment:
I am sorry to nitpick, but there still are 8 occurrences of `array()` in this PR